CorsixTH/Lua/world.lua

```diff
@@ -544,14 +554,14 @@ end
 
 function World:createEarthquake()
   -- Sanity check
-  if not self.earthquake_size then
+  if not self.next_earthquake or not self.next_earthquake.size then
```

Comment:
Can't test active here, active is expected to be false. start_day and start_month should be nil or 0 if the earthquake shouldn't happen though so the size test is unnecessary.